app/Services/Model/AttendeeService.php

```diff
@@ -173,6 +183,19 @@ public function addAttendee(Summit $summit, array $data)
 
             $attendee = SummitAttendeeFactory::build($summit, $data, $member);
 
+            // tags
+            if (isset($data['tags'])) {
+                $attendee->clearTags();
+                foreach ($data['tags'] as $val) {
+                    $tag = $this->tag_repository->getByTag($val);
+                    if ($tag == null) {
```

Comment:
lets move this to a private method like
populateTags(attendee, $payload):Attendee to avoid code dupe

Reply:
done